README.md

```diff
@@ -1,24 +1,78 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
 
-Things you may want to cover:
+> This web application is a Ruby on Rails application that allows you to manage your budget in a professional way. You can register and/or log in and your data will be safely saved and you will be the only one person to manage access them. You can perform transaction transactions associated with a category of your transaction then at the end of the day you can see the money spent on each category.
 
-* Ruby version
 
-* System dependencies
+## Built With
 
-* Configuration
+- Major: Ruby, HTML,CSS
+- Frameworks: Rails, Bootstrap5
+- Libraries: Cancancan, Devise
+- Technologies: Git, Gitflow, Linters, Github, PostgreSql
 
-* Database creation
+## Live Demo (not available yet)
```

Comment:
- [ ] **Optional:** The link is now available, kindly remove the (not available yet) in the **Live Demo** Section title.